syntax = "proto3";
package RSOAExample;

message ValueReq
{
    uint32 id = 1;
}